app/jobs/psql_stats_job.rb

```diff
@@ -0,0 +1,81 @@
+class PsqlStatsJob < ApplicationJob
+  queue_as :default
+
```

Comment:
we should probably add a concurrency key to this so only one job at runs every minute, otherwise all 18 hosts or whatever will enqueue a copy

https://github.com/18F/identity-idp/blob/6891d1c09d1402477351fc4b186bb8942875965f/app/jobs/phone_number_opt_out_sync_job.rb#L3-L11

Reply:
yikes, good catch @zachmargolis!!! I didn't catch that with only one worker working.

Reply:
This should be resolved, can you confirm?